README.md

```diff
@@ -99,7 +99,7 @@ func main() {
 > ```shell
```

Comment:
Consider making the intro paragraph more consistent with the example Node Readme: https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk

Reply:
Done!